envoy/http/api_listener.h

```diff
@@ -5,6 +5,22 @@
 namespace Envoy {
 namespace Http {
 
+class RequestDecoderHandle {
+public:
+  virtual ~RequestDecoderHandle() = default;
+
+  /**
+   * @return true if the underlying decoder is still valid.
+   */
+  virtual bool isValid() PURE;
```

Comment:
I think this method is never called. Maybe we can remove it?

Reply:
Good point! Done.